packages/@angular/cli/lib/config/schema.json

```diff
@@ -292,6 +292,10 @@
       },
       "additionalProperties": false
     },
+    "packageManager": {
+        "enum": [ "npm", "cnpm", "yarn", "default" ],
+        "default": "npm"
```

Comment:
@hansl I tried but then it always picks value as default and it fails install as there is nothing for  `default install`. Am I missing how it should be done ?